db/migrate/20200324170819_add_welcome_notifications_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddWelcomeNotificationsToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :welcome_notifications, :boolean, default: true, null: false
```

Comment:
`user.welcome_notifications` seems to me like it would return an active record relation of notifications rather than a boolean. What about something more boolean like `user.subscribed__to_welcome_notifications` or something

Reply:
@mstruve I agree with you here, but all of the other boolean flags on the user are named the same way (`mod_roundrobin_notifications`, `email_follower_notifications`, etc.). It seemed better to follow the convention set rather than diverge from it here.

That being said, I do think we should rename all of these columns at some point (cool down period?), but it definitely feels out of scope for this PR.

Reply:
Okie dokie!

Reply:
@mstruve Created #6892 to make sure that this change doesn't get forgotten during the cool down period :)